### ISSUE TITLE
Can I get a little filtering in my life?

### ISSUE THREAD

**Issue by jaredgib** [thunkable/thunkable-issues#479]
**Is your feature request related to a problem? Please describe.**
I'm frustrated when my users have to filter data that doesn't belong to them when I use airtable. 

**Describe the solution you'd like**
I want to use Airtables own 'filter by formula' to do this for me when i send my request, So instead of "get all rows" and then filtering on the phone. 

I want to:

Get all rows where property (Example) (=) or (contains) ('sampleCriteria')

**Describe alternatives you've considered**
I filter all data on the user's phone. this leads to a slowdown in processing time. My database can hold up to 50000 records... think about that if it was full but the user only has 25 records. Airtable will do this filtering for me. 

I do this myself right now, but a block setup would be cleaner :) and would further support the noCode movement ;) 

**Which platform are you asking for?**
Thinkable X

**Additional context**

Here is a working example
https://community.thunkable.com/t/solved-spreadsheet-query/286894/10?u=jared

**Comment by easaw** [comment]
Can't you do this with Airtable WebAPI?

I create custom post types in WordPress and use the API URL with a filter, so I only get the post(s) needed. Then i get a familiar UI that controls a database (fine for small datasets).

Of course Google Sheets is now an option with the new advanced lists (in Beta) and obviously has huge cost advantages over Airtable.

**Comment by jaredgib** [comment]
> Can't you do this with Airtable WebAPI?
> 
> Of course Google Sheets is now an option with the new advanced lists (in Beta) and obviously has huge cost advantages over Airtable.

Yes. That is what is in my example above. This requires the user understanding the api documentation. It also results in the user having to be savvy enough to decode the Json response, look for the indicator for pagination occurring, and then to retrieve the next 100 results.

Airtable api Filtering doesn’t return all records that match criteria. Only 100 at a time. This is where things are harder for a user. The api call goes into a loop of there is the “more than 100 records key” present in the original results.

**Comment by kezb765** [comment]
Hi jared,

thanks for your advice, I found my bug and closed #518 because the airtable filter now works again:

![grafik](https://user-images.githubusercontent.com/7669660/82328247-e65f8d80-99df-11ea-9a72-8a253158b09c.png)
